packages/typescript/test/ts_project/swc/BUILD.bazel

```diff
@@ -31,10 +40,17 @@ ts_project(
         args = ["--env-name=test"],
         swcrc = "//:.swcrc",
     ),
-    tsconfig = {
-        "compilerOptions": {
-            "declaration": True,
-            "sourceMap": True,
-        },
-    },
+    tsconfig = _TSCONFIG,
+)
+
+ts_project(
+    name = "transpile_with_typeerror",
+    srcs = ["typeerror.ts"],
+    # The transpile_with_typeerror.check target will have a build failure
```

Comment:
How does the build failure get ignored on CI with this one?

Reply:
the manual tag gets propagated to the build targets, so we don't run them. The test explicitly asks for the js_library which passes because typecheck action doesn't run

Reply:
🤦  I was starring right at it and didn't see the `"manual"` tag